components/automate-cli/pkg/verifyserver/routes.go

```diff
@@ -0,0 +1,20 @@
+package verifyserver
+
+import (
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/handlers"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/logger"
+	fiber_utils "github.com/chef/automate/components/automate-cli/pkg/verifyserver/utils/fiber"
+	"github.com/gofiber/fiber"
+)
+
+func SetupRoutes(app *fiber.App, h *handlers.Groups, logger logger.ILogger) {
+	// Status
+	app.Get("/status", h.Status.GetStatus)
```

Comment:
we can directly call the handler here. I don't see any usage of having interfaces for handlers.